src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed === true,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      {todo.user && <UserInfo user={todo.user} />}
```

Comment:
[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has null or undefined value. In this case, you are rendering the UserInfo component even if todo.user is null or undefined. Make sure to check if the property is not null or undefined before rendering the component.